stage1/modules/self-introduction/README.md

```diff
@@ -1,70 +1,69 @@
 
-> ### Важно
+> ### Important
 >
-> - Проверьте, что видео и текст доступны по ссылкам и для просмотра не требуется никаких дополнительных разрешений. Проще всего проверить это, открыв обе ссылки в режиме инкогнито в браузере.
-> - Мы не просим вас загружать видео в открытый доступ. Вы можете [включить доступ к ролику только по ссылке](#работа-с-youtube). Такое видео смогут просмотреть только пользователи, у которых есть нужный URL.
+> - Ensure that the video and transcript can be accessed through the links you provide and do not require any additional permissions. The easiest way to check this is to try opening both links in Incognito mode in your browser.
+> - We do not require you to make the video public. You can make it [accessible only via a link](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en). This way only the users who receive the correct URL from you will be able to watch the video.
 
-## Проверка задания
+## Task Check
 
-Факт записи видеоролика будет проверяться с помощью [кросс-чека](https://docs.rs.school/#/cross-check-flow), а его содержание будут оценивать менторы.
+During [Cross-Check](https://docs.rs.school/#/cross-check-flow) you will have to check the fact that there is a recording, but **not** its content. The content will later be assessed by mentors.
 
-Чтобы отправить выполненное задание на проверку, зайдите в [rs app](https://app.rs.school/), перейдите на вкладку Cross-Check: Submit, выберите задание **Self-Introduction** и добавьте в форму **ссылку на деплой**.
+To submit the completed task for review, go to [RS App](https://app.rs.school/), choose the Cross-Check: Submit tab, select the **Self-Introduction** task in the dropdown menu, and add your **deployment link** to the form.
 
-Также не забудьте сделать Pull Request из ветки разработки в `Main` (этот PR не мержить), чтобы презентацию смог посмотреть ментор.
+Also, remember to create a Pull Request from the development branch `self-introduction` to `main` (do not merge this PR) for mentors to be able to watch your presentation.
 
-### Критерии оценки
+### Assessment Criteria
 
-Максимальный балл за задание — **100 баллов**
+The maximum score for the task is **100 points**.
 
-- Есть ссылка на видеозапись выступления. В видео автор рассказывает о себе на английском языке, видео имеет длительность около 3–5 минут (отклонение в пределах 15 секунд ошибкой считать не нужно) — **+50 баллов**
-- Есть транскрипт выступления. Транскрипт на английском языке и содержит полный текст выступления (может быть незначительно отредактирован) — **+50 баллов**
+- In the video, the author talks about themselves in English, and the video lasts for about 3-5 minutes (variation within 15 seconds must not be considered either an error or a reason to subtract any points) — **+50 points**.
```

Comment:
I think it's better not to remove this part "There is a link to the video presentation."

Reply:
Done ✅